src/mainwindow.py

```diff
@@ -50,7 +50,8 @@ def __init__(self, app):
         self.old_versions = []
         self.printing = html2.PrintingSystem(self)
 
-        self.aui = aui.AuiManager(self)
+        extra_flags = aui.AUI_MGR_LIVE_RESIZE

```

Comment:
Should this be `aui.AUI_MGR_DEFAULT | aui.AUI_MGR_LIVE_RESIZE` to preserve any default flags?

Reply:
Maybe. It seems to behave the same either way but I added it in case.